astropy/coordinates/tests/test_frames_with_velocity.py

```diff
@@ -1,36 +1,37 @@
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
-
 import pytest
 import numpy as np
 
 from astropy import units as u
 from astropy.coordinates.builtin_frames import ICRS, Galactic, Galactocentric
 from astropy.coordinates import builtin_frames as bf
+from astropy.coordinates import galactocentric_frame_defaults
 from astropy.units import allclose as quantity_allclose
 from astropy.coordinates.errors import ConvertError
 from astropy.coordinates import representation as r
 
 
 def test_api():
     # transform observed Barycentric velocities to full-space Galactocentric
-    gc_frame = Galactocentric()
-    icrs = ICRS(ra=151.*u.deg, dec=-16*u.deg, distance=101*u.pc,
-                pm_ra_cosdec=21*u.mas/u.yr, pm_dec=-71*u.mas/u.yr,
-                radial_velocity=71*u.km/u.s)
-    icrs.transform_to(gc_frame)
+    with galactocentric_frame_defaults.set('pre-v4.0'):
```

Comment:
Is this the correct value?

Reply:
I'm going to guess that it might make more sense to use 'latest' here to make sure this example always works with the latest version (since we're not checking output values here, just that it runs).

Reply:
Done!